File: Shared/Students.Models/ReferenceModels/EducationForm.cs
namespace Students.Models.ReferenceModels;

/// <summary>
/// Форма обучения
/// </summary>
public class EducationForm
{
  /// <summary>
  /// Id формы
  /// </summary>
  public Guid Id { get; set; }

  /// <summary>
  /// Имя формы обучения
  /// </summary>
  public string? Name { get; set; }
}

File: Client/src/storage/catalogConfigs/educationForm.js
import {
    useGetAllAsync,
    useGetAllPagedAsync,
    useGetOneByIdAsync,
    useAddOneAsync,
    useEditOneAsync,
    useRemoveOneAsync,
} from '../crud/educationFormCrud.js';
import { educationFormModel } from '../models/index.js';

export default {
    detailsLink: 'educationForm',
    hasDetailsPage: false,
    serverPaged: false,
    properties: educationFormModel,
    crud: {
        useGetAllAsync,
        useGetAllPagedAsync,
        useGetOneByIdAsync,
        useAddOneAsync,
        useEditOneAsync,
        useRemoveOneAsync,
    },
    columns: [
        {
            title: 'Форма образования',
            dataIndex: 'name',
            key: 'name',
        },
    ],
    dataConverter: (data) => data,
};